app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewsController < ApplicationController
+  before_filter :require_user
+  
+  def create
+    @video = Video.find(params[:video_id])
+    review = @video.reviews.build(review_params)
+
+    if review.save
+      redirect_to @video
+    else
+      @reviews = @video.reviews.reload
+      render 'videos/show'
+    end
+  end
+
+  private
+
+  def review_params
+    params.require(:review).merge!(user: current_user).permit!
```

Comment:
Using `permit!` is not a good way, because you allow all attributes to be set by user. This is never used in real app for security reasons.